internals/overlord/servstate/request_test.go

```diff
@@ -18,13 +18,73 @@ import (
 	. "gopkg.in/check.v1"
 
 	"github.com/canonical/pebble/internals/overlord/servstate"
+	. "github.com/canonical/pebble/internals/testutil"
 )
 
 func (s *S) TestStart(c *C) {
+	s.newServiceManager(c)
+	layer := `
+services:
+    one:
+        override: replace
+        command: /bin/sh -c "echo one; sleep 10"
+        startup: enabled
+
+    two:
+        override: replace
+        command: /bin/sh -c "echo two; sleep 10"
+        startup: enabled
+`
+	s.planAddLayer(c, layer)
+	s.planChanged(c)
+
+	s.st.Lock()
+	defer s.st.Unlock()
+
+	tset, err := servstate.Start(s.st, [][]string{{"one"}, {"two"}})
+	c.Assert(err, IsNil)
+
+	tasks := tset.Tasks()
+	c.Assert(len(tasks), Equals, 2)
+
+	c.Assert(tasks[0].Kind(), Equals, "start")
+	req, err := servstate.TaskServiceRequest(tasks[0])
+	c.Assert(err, IsNil)
+	c.Assert(req.Name, Equals, "one")
+
+	c.Assert(tasks[1].Kind(), Equals, "start")
+	req, err = servstate.TaskServiceRequest(tasks[1])
+	c.Assert(err, IsNil)
+	c.Assert(req.Name, Equals, "two")
+
+	c.Assert(tasks[0].Lanes()[0], IntNotEqual, tasks[1].Lanes()[0])
```

Comment:
How is `IntNotEqual` different from `Not(Equals)`? I've never used that one.

Reply:
I did some testings and it seems they are the same, at least in this case.

I also searched the repo, and although there are occurrences of `IntNotEqual`, there are more occurrences of `Not(Equals)`, so I will change it to the latter for consistency.